builtin/mainmenu/tab_online.lua

```diff
@@ -84,13 +86,13 @@ local function get_formspec(tabview, name, tabdata)
 		"label[0.25,0.35;" .. fgettext("Address") .. "]" ..
 		"label[4.25,0.35;" .. fgettext("Port") .. "]" ..
 		"field[0.25,0.5;4,0.75;te_address;;" ..
-			core.formspec_escape(core.settings:get("address")) .. "]" ..
+		core.formspec_escape(core.settings:get("address")) .. "]" ..
```

Comment:
Removing the indentation here is also a change for the worse IMO. Please check your editor configuration to make sure it isn't running some kind of aggressive auto formatter.